### PR TITLE
update documentation

### PR DESCRIPTION
**Pull request by n-h-diaz**
add documentation for relational tables

### CODE REVIEW

File: csv_download.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+title: CSV Download
+nav_order: 6
+has_children: false
+---
+
+# CSV Download
+
+We provide access to some of our data in a relational format, which is available for CSV download in a [public GCS bucket](https://console.cloud.google.com/storage/browser/relational_tables). The tables are constructed such that each row represents a [Place](https://datacommons.org/browser/Place) and each column represents a [Statistical Variable](https://datacommons.org/browser/StatisticalVariable).
```
Comment (beets) [inline csv_download.md:10]:
```suggestion
We provide access to some of our data in a relational format, which is available for CSV download in a [public Google Cloud Storage bucket](https://console.cloud.google.com/storage/browser/relational_tables). The tables are constructed such that each row represents a [Place](https://datacommons.org/browser/Place) and each column represents a [Statistical Variable](https://datacommons.org/browser/StatisticalVariable).
```

File: csv_download.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+The public bucket is organized into different vertical folders:
```
Comment (beets) [inline csv_download.md:12]:
```suggestion
The public bucket is organized by vertical, each in a different folder:
```

File: csv_download.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+* [Housing](https://console.cloud.google.com/storage/browser/relational_tables/housing)
+
+Each vertical folder contains tables for various Place categories: `all` (all places), `us` (US places), `non_us` (non-US places), `county` (US counties), and `zip` (US zip codes). For each vertical and Place category, there are three types of tables:
+* `value`: Each cell contains the value of the latest observation for a given Statistical Variable and Place.
+* `date`: Each cell contains the date of the latest observation for a given Statistical Variable and Place.
+* `provenance`: Each cell contains the provenance of the latest observation for a given Statistical Variable and Place, as well as the [measurement method](https://datacommons.org/browser/measurementMethod), if provided. Measurement methods that are prefixed with `dcAggregate/` represent Data Commons aggregated values.
+
+The corresponding `value`, `date`, and `provenance` tables can be joined using the first three columns, which contain information about the place:
+* `place_name`: The name(s) of the Place.
+* `place_dcid`: The [Data Commons ID](https://datacommons.org/browser/dcid) for the Place.
```
Comment (beets) [inline csv_download.md:26]:
```suggestion
* `place_dcid`: The [Data Commons ID](https://docs.datacommons.org/glossary.html) for the Place.
```

File: csv_download.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+* [Education](https://console.cloud.google.com/storage/browser/relational_tables/education)
+* [Employment](https://console.cloud.google.com/storage/browser/relational_tables/employment)
+* [Health](https://console.cloud.google.com/storage/browser/relational_tables/health)
+* [Housing](https://console.cloud.google.com/storage/browser/relational_tables/housing)
+
+Each vertical folder contains tables for various Place categories: `all` (all places), `us` (US places), `non_us` (non-US places), `county` (US counties), and `zip` (US zip codes). For each vertical and Place category, there are three types of tables:
+* `value`: Each cell contains the value of the latest observation for a given Statistical Variable and Place.
+* `date`: Each cell contains the date of the latest observation for a given Statistical Variable and Place.
+* `provenance`: Each cell contains the provenance of the latest observation for a given Statistical Variable and Place, as well as the [measurement method](https://datacommons.org/browser/measurementMethod), if provided. Measurement methods that are prefixed with `dcAggregate/` represent Data Commons aggregated values.
+
```
Comment (beets) [inline csv_download.md:23]: also add a note for how the file names are formatted for these tables, and that they are sharded.

File: csv_download.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+* [Education](https://console.cloud.google.com/storage/browser/relational_tables/education)
+* [Employment](https://console.cloud.google.com/storage/browser/relational_tables/employment)
+* [Health](https://console.cloud.google.com/storage/browser/relational_tables/health)
+* [Housing](https://console.cloud.google.com/storage/browser/relational_tables/housing)
+
+Each vertical folder contains tables for various Place categories: `all` (all places), `us` (US places), `non_us` (non-US places), `county` (US counties), and `zip` (US zip codes). For each vertical and Place category, there are three types of tables:
+* `value`: Each cell contains the value of the latest observation for a given Statistical Variable and Place.
+* `date`: Each cell contains the date of the latest observation for a given Statistical Variable and Place.
+* `provenance`: Each cell contains the provenance of the latest observation for a given Statistical Variable and Place, as well as the [measurement method](https://datacommons.org/browser/measurementMethod), if provided. Measurement methods that are prefixed with `dcAggregate/` represent Data Commons aggregated values.
```
Comment (beets) [inline csv_download.md:22]: please also describe how both are represented in the same cell (looks like `|` separated)

Comment (beets) [reply csv_download.md:22]:
```suggestion
* `provenance`: Each cell contains the provenance of the latest observation for a given Statistical Variable and Place, as well as the [measurement method](https://docs.datacommons.org/glossary.html), if provided. Measurement methods that are prefixed with `dcAggregate/` represent Data Commons aggregated values.
```

File: csv_download.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+The corresponding `value`, `date`, and `provenance` tables can be joined using the first three columns, which contain information about the place:
+* `place_name`: The name(s) of the Place.
+* `place_dcid`: The [Data Commons ID](https://datacommons.org/browser/dcid) for the Place.
+* `place_type`: The type(s) of the Place.
```
Comment (beets) [inline csv_download.md:27]: would be nice to also have an example of the contents of the files, maybe with a couple columns then ellipsis?

File: csv_download.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+title: CSV Download
+nav_order: 6
+has_children: false
+---
+
+# CSV Download
```
Comment (beets) [inline csv_download.md:8]: maybe CSV Bulk Download? or Relational Tables

**Review by beets — COMMENTED** [review]
thanks natalie! added comments, thinking about this from a newcomers perspective.

File: csv_download.md (hunk 4 above)
Comment (n-h-diaz) [reply csv_download.md:25]: added

File: csv_download.md (hunk 7 above)
Comment (n-h-diaz) [reply csv_download.md]: changed to CSV Bulk Download

File: csv_download.md (hunk 6 above)
Comment (n-h-diaz) [reply csv_download.md:29]: added a sample table as example

**Comment by n-h-diaz** [comment]
thanks for the review! :)

**Review by beets — APPROVED** [review]
Sorry - these comments didn't go through!

File: csv_download.md [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+The table names follow the pattern `[vertical]_[place_category]_[type]` and are sharded into multiple CSV files. (For example, the file [`demographics_all_date-00000-of-00456.csv`](https://storage.googleapis.com/relational_tables/demographics/demographics_all_date-00000-of-00456.csv) contains a portion of the observation `dates` for `demographics` Statistical Variables and `all` Places.)
+
+The corresponding `value`, `date`, and `provenance` tables can be joined using the first three columns, which contain information about the place:
+* `place_name`: The name(s) of the Place.
+* `place_dcid`: The [Data Commons ID](https://docs.datacommons.org/glossary.html) for the Place.
+* `place_type`: The type(s) of the Place.
+
+## Example Table Structure
+
+Below is a subset of the `housing_county_value` table:
```
[the remark this answers is not in the document]
Comment (n-h-diaz) [reply csv_download.md:35]: done

File: csv_download.md [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+
+Each vertical folder contains tables for various Place categories: `all` (all places), `us` (US places), `non_us` (non-US places), `county` (US counties), and `zip` (US zip codes). For each vertical and Place category, there are three types of tables:
+* `value`: Each cell contains the value of the latest observation for a given Statistical Variable and Place.
+* `date`: Each cell contains the date of the latest observation for a given Statistical Variable and Place.
+* `provenance`: Each cell contains the provenance of the latest observation for a given Statistical Variable and Place, as well as the [measurement method](https://docs.datacommons.org/glossary.html), if provided. Measurement methods that are prefixed with `dcAggregate/` represent Data Commons aggregated values.
+
+The table names follow the pattern `[vertical]_[place_category]_[type]` and are sharded into multiple CSV files. (For example, the file [`demographics_all_date-00000-of-00456.csv`](https://storage.googleapis.com/relational_tables/demographics/demographics_all_date-00000-of-00456.csv) contains a portion of the observation `dates` for `demographics` Statistical Variables and `all` Places.)
```
[the remark this answers is not in the document]
Comment (n-h-diaz) [reply csv_download.md]: done

**Comment by n-h-diaz** [comment]
thanks for the review!

**Pull request merged**